Pipfile

```diff
@@ -16,4 +18,5 @@ mypy = '*'
 python_version = '3.7'
 
 [scripts]
-upload = 'python -m vidpub'
+upload = 'python -m vidpub --action=upload'
+generate = 'python -m vidpub --action=generate'
```

Comment:
Would `python -m vidpub upload` look better?

Also `generate` is not a good name. Maybe `python -m vidpub playlist`? (Since this aligns better to what the function is *actually* doing.

Reply:
Done!